compiler/src/dmd/dmodule.d

```diff
@@ -499,20 +500,26 @@ extern (C++) final class Module : Package
 
     static const(char)* find(const(char)* filename)
     {
-        return find(filename.toDString).ptr;
+        ImportPathInfo pathThatFoundThis; // is this needed? In fact is this function needed still???
```

Comment:
LDC/GDC, likely yes.

Reply:
I couldn't find an example of either.

I was going to remove it, but thought better of it. Best to leave a comment that it is possibly dead code.

Reply:
Hmm, indeed 
```
$ git grep Module\.find
dmd/dmodule.d:        if (Module.find(getFilename(packages, ident)))
```
not sure about GDC.